tokio-timer/src/timeout.rs

```diff
@@ -286,18 +285,6 @@ impl<T> Error<T> {
     }
 }
 
-impl<T: error::Error> error::Error for Error<T> {
```

Comment:
Why remove this?

Reply:
Bacause the `description` function of `std::error::Error` has been deprecated.

https://doc.rust-lang.org/std/error/trait.Error.html#method.description